ui/css/design-system/_z-index.scss

```diff
@@ -2,13 +2,9 @@
   Z-Indicies
 */
 $dropdown-z-index: 30;
-$token-icon-z-index: 15;
-$container-z-index: 15;
-$header-z-index: 12;
-$mobile-header-z-index: 26;
 $main-container-z-index: 18;
-$send-card-z-index: 20;
-$sidebar-z-index: 26;
-$sidebar-overlay-z-index: 25;
+$mobile-header-z-index: 26;
+$toast-z-index: 200;
+$header-z-index: 300;
 $modal-z-index: 1050;
```

Comment:
Checked for removed z-indexes

<img width="200" alt="Screenshot 2025-05-23 at 1 14 33 PM" src="https://github.com/user-attachments/assets/228fa863-2e01-4634-91f2-34a993628d0f" /><img width="200" alt="Screenshot 2025-05-23 at 1 15 17 PM" src="https://github.com/user-attachments/assets/3ddb85aa-8d54-415a-96af-49273469b512" /><img width="200" alt="Screenshot 2025-05-23 at 1 15 40 PM" src="https://github.com/user-attachments/assets/cfdf79bc-9ba4-477b-a8b2-963fbe5211a8" /><img width="200" alt="Screenshot 2025-05-23 at 1 15 51 PM" src="https://github.com/user-attachments/assets/6836d6dc-56b3-478f-a079-b3c0390494e5" /><img width="200" alt="Screenshot 2025-05-23 at 1 16 01 PM" src="https://github.com/user-attachments/assets/cbd01af1-d2e3-4335-882b-4e505b262801" />